### PR TITLE
Fixed issue where some input generates codes with NaN instead of null

### PR DESCRIPTION
**Pull request by PaddyCo**
Codes with non-existing elementary color specified (4055-**Z**95B for example) would not return null but return `rgb(NaN,NaN,NaN)` or `#NaNNaNNaN` instead

Corrected the Regex and added a simple test for this specific case 😄 

@m90

### CODE REVIEW

File: ncscolor.js [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 })(this, function(){
 
-	var ncsRe = /^(?:NCS|NCS\sS)\s(\d{2})(\d{2})-(N|[A-Z])(\d{2})?([A-Z])?$/;
+	var ncsRe = /^(?:NCS|NCS\sS)\s(\d{2})(\d{2})-(N|R|G|B|Y)(\d{2})?([R|G|B|Y])?$/;
```
Comment (m90) [inline ncscolor.js]: I guess the second `R|G|B|Y` doesn't need to be in a `[]`-range anymore now, just like in the preceding part?

File: test.js [hunk 2]
```diff
@@ -19,7 +19,9 @@ describe('ncs', function(){
 		assert.equal(ncs.rgb(''), null);
 		assert.equal(ncs.hex('NCS5000-N'), null);
 		assert.equal(ncs.hex('12345'), null);
+		assert.equal(ncs.hex('NCS 5000-Z20Y'), null);
```
Comment (m90) [inline test.js]: Out of the scope of this PR, but could you make it so that all the `null` tests use `strictEqual`? That would probably be handy to catch even more possible edge cases.

**Review by m90 — COMMENTED** [review]
I left two tiny remarks, other than that :100: thanks a bunch!

**Comment by PaddyCo** [comment]
Thanks for the quick response, and for the very handy module 🙌

**Comment by m90** [comment]
This is now on npm at version `1.0.0` (major version bump as I dropped old Node versions).
